app/src/main/kotlin/com/intive/picover/parties/repository/PartiesRepository.kt

```diff
@@ -20,6 +21,16 @@ class PartiesRepository @Inject constructor(
 			}
 		}
 
+	suspend fun addParty(partyRemote: PartyRemote) =
+		try {
```

Comment:
`runCatching` adds `Result` wrapper out of the box :)

Reply:
@bartek977 Really good point. thank you. Done.